beamer/agent/chain.py

```diff
@@ -572,6 +575,61 @@ def _timestamp_is_l1_finalized(
     return int(time.time()) > timestamp + target_chain_finality
 
 
+def maybe_prove(claim: Claim, context: Context) -> bool:
```

Comment:
Why does this function return `bool`, or anything? The return value does not seem to be used anywhere.